main.go

```diff
@@ -43,7 +57,5 @@ func main() {
 		logger.Error("failed to run the script", zap.String("arg", args[1]), zap.Error(err))
 	}
 
-	if err := logger.Sync(); err != nil {
-		panic(err)
-	}
+	_ = logger.Sync()
```

Comment:
why this change ?

Reply:
this was removed as `logger.Sync()` returns an error whenever we log an error from Zap, which causes panic.
You can read more about it here
https://github.com/uber-go/zap/issues/328#fromHistory
https://github.com/uber-go/zap/pull/347